examples/lush_tutorial.lua

```diff
@@ -91,12 +84,10 @@ local sea_deep  = hsl(208, 90, 10)  -- you can just type them normally.
 -- Try editing these values to see the colors update in real time.
 
 -- You can also give RGB colors to hsl() and it will convert them into the HSL
--- colorspace. Note this can be slightly lossy due to transatling between the
--- two colorspaces.
+-- colorspace. Note that this can be slightly lossy due to the translation between 
+-- the two colorspaces.
```

Comment:
Perhaps we can actually write this whole part better. Something like:

You can also pass RGB colors to hsl() in the form "#RRGGBB". They will be automatically converted into hue, saturation and lightness values but be warned that translating between colorspaces is naturally inaccurate and different tools will give different translations. Your colors may need some manual adjustment afterwards.